duckduckgo/iOS
# Widget Education

Task/Issue URL: https://app.asana.com/0/72649045549333/1200189110180410/f

**Description**:
Introduce a new Home screen message that informs users on how to install the app widget within iOS system.

**Steps to test this PR**:
1. Make sure you run at least iOS 14.0.
2. Since the feature is part of the experiment make sure to comment out the variantManager code in HomeMessageStorage and in Settings.
3. You should be able to see the home message on the Home screen.
4. Once entered you can see the modal with instructions on how to add the app widget.
5. Same screen can be seen from the Settings under the "Add Widget to Home Screen" cell.
3. Once dismissed it should never appear again.

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:

* [ ] iOS 13 (the feature shouldn't be visible there)

**Theme Testing**:
